### PR TITLE
ci: Bump release and publish to Python 3.9 build

### LINKED ISSUE #48: Update release and publish to Python 3.9
Consider to update release and publish to Python 3.9. Current release process runs with Python 3.7

### PR DESCRIPTION
**Pull request by mortendaehli**
### WHAT

Bump GitHub Actions workflows to release and publish using Python 3.9. Up from Python 3.7.

### WHY

Testing for multiple versions and OS. We would like to build the release using the current stable Python release which is Python 3.9/3.10.

### HOW

Change pipeline yaml-files.

### What to test/verify

Not needed. Already building Python 3.9.
---

### Contributor checklist

- [x] :tada: This PR closes #48 .
- [x] :scroll: I have broken down my PR into the following tasks:
   - [x] Task 1: Change from Python 3.7 to 3.9 in workflow yaml-files.
- [x] :page_with_curl: The following issues are related:
   - [x] #31

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -15,10 +15,10 @@ jobs:
           fetch-depth: 0
 
       # We run the  oldest LTS version for now: https://endoflife.date/python
```
Comment (HaraldGSvendsen) [inline .github/workflows/publish.yml:17]: outdated comment?

**Review by HaraldGSvendsen — APPROVED** [review]
OK

File: .github/workflows/publish.yml (hunk 1 above)
Comment (mortendaehli) [reply .github/workflows/publish.yml]: Thanks! Removed it now.

**Review by HaraldGSvendsen — APPROVED** [review]
OK

**Pull request merged**